drupal: add page (#3589)
tldr-pages/tldr: pages/common/drupal.md

# drupal

> > Drupal Console.

Reviewer: Perhaps this will read better?
```suggestion
> CLI for Drupal.
```

> > The Drupal CLI. A tool to generate boilerplate code, interact with and debug Drupal.

Reviewer: Perhaps this will read better?
```suggestion
> Generates boilerplate code, interacts with and debugs Drupal projects.
```

> More information: <https://drupalconsole.com/>.

> - Install module "foo":

Reviewer 2: Perhaps it's better to say "Install module:" and in the example, change it to "module_name" from "foo". And do the same for next one too.

Author: I tried to use the same syntax as the `drush` page. But I agree `{{module_name}}` is easier to understand.

`drupal module:install {{foo}}`

- Uninstall module "foo":

`drupal module:uninstall {{foo}}`

- Clear all caches:

`drupal cache:rebuild`

> - View current Drupal Installation status:

Reviewer 3: Should `**I**nstallation` be **i**nstallation here?

Reviewer 2: Yes, I think so.
